### PR TITLE
[Core] Docstring full stops and a few other grammar fixes

### PR DESCRIPTION
**Pull request by Vexed01**
### Type

- [x] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
It seems that the majority of the docstrings in core_commands.py don't have full stops - this adds them!
There are also a few general grammar fixes as well.

### CODE REVIEW

File: redbot/core/core_commands.py [hunk 1]
```diff
@@ -2357,16 +2358,16 @@ async def command_disabledmsg(self, ctx: commands.Context, *, message: str = "")
...
         pass
 
     @autoimmune_group.command(name="list")
     async def autoimmune_list(self, ctx: commands.Context):
         """
-        Get's the current members and roles
+        Gets the current members and roles.
 
-        configured for automatic moderation action immunity
+        Configured for automatic moderation action immunity.
```
Comment (aikaterna) [inline redbot/core/core_commands.py]: This is one string here: "Gets the current members and roles configured for automatic moderation action immunity."

File: redbot/core/core_commands.py [hunk 2]
```diff
@@ -1428,7 +1429,7 @@ async def regionalformat(self, ctx: commands.Context, language_code: str = None)
     @_set.command()
     @checks.is_owner()
     async def custominfo(self, ctx: commands.Context, *, text: str = None):
-        """Customizes a section of [p]info
+        """Customizes a section of [p]info.
```
Comment (Jackenmen) [inline redbot/core/core_commands.py]: Quotation needed, but I believe we decided that command names in help docstrings in Red should be put in backquotes. (don't change this until someone confirms)
```suggestion
        """Customizes a section of `[p]info`.
```

File: redbot/core/core_commands.py [hunk 3]
```diff
@@ -2465,7 +2465,7 @@ async def adddestination(
         self, ctx: commands.Context, *, channel: Union[discord.TextChannel, int]
     ):
         """
-        Adds a destination text channel to recieve owner notifications
+        Adds a destination text channel to recieve owner notifications.
```
Comment (Jackenmen) [inline redbot/core/core_commands.py]:
```suggestion
        Adds a destination text channel to receive owner notifications.
```

File: redbot/core/core_commands.py [hunk 4]
```diff
@@ -2407,7 +2407,7 @@ async def autoimmune_remove(
         self, ctx: commands.Context, *, user_or_role: Union[discord.Member, discord.Role]
     ):
         """
-        Makes a user or roles immune from automated moderation actions
+        Makes a user or roles immune from automated moderation actions.
```
Comment (Jackenmen) [inline redbot/core/core_commands.py]:
```suggestion
        Makes a user or role immune from automated moderation actions.
```

File: redbot/core/core_commands.py [hunk 5]
```diff
@@ -2394,7 +2394,7 @@ async def autoimmune_add(
...
```
Comment (Jackenmen) [inline redbot/core/core_commands.py]:
```suggestion
        Makes a user or role immune from automated moderation actions.
```

File: redbot/core/core_commands.py (hunk 2 above)
Comment (Jackenmen) [reply redbot/core/core_commands.py]: You know... Just do it 👍

File: redbot/core/core_commands.py [hunk 6]
```diff
@@ -157,7 +157,7 @@ async def _load(
 
     @staticmethod
     def _cleanup_and_refresh_modules(module_name: str) -> None:
-        """Interally reloads modules so that changes are detected"""
+        """Interally reloads modules so that changes are detected."""
```
Comment (Jackenmen) [inline redbot/core/core_commands.py]:
```suggestion
        """Internally reloads modules so that changes are detected."""
```

**Review by Jackenmen — CHANGES_REQUESTED** [review]
Looks good, but I found few typos in the lines edited by this PR that could be addressed by it as well

**Review by Jackenmen — APPROVED** [review]
Looks good to me.

**Pull request merged**